forge-gui/res/cardsfolder/upcoming/attempted_murder.txt

```diff
@@ -0,0 +1,11 @@
+Name:Attempted Murder
+ManaCost:X B B
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature | IsCurse$ True | SubAbility$ DBRepeat | SpellDescription$ Choose target creature. Roll X six-sided dice. For each even result, put two -1/-1 counters on that creature. For each odd result, create a 1/2 blue Bird creature token with flying named Storm Crow.
+SVar:DBRepeat:DB$ Repeat | MaxRepeat$ X | RepeatSubAbility$ DBRollDice
```

Comment:
You can't do this with Repeat because of "Whenever you roll one or more dice" would trigger multiple times

Reply:
Hmm I tried with Rolldice | Amount X but that didn't work . 

Would it need engine changes ?

Reply:
Yea but shouldn't be too bad

Currently RollDiceEffect will check the total (sum of all rolls) so you just need to add another loop that iterates over `rolls` and a new param to enable to behaviour

Reply:
Ok will close this PR then